Sources/App/Views/PackageController/GetRoute.Model+ext.swift

```diff
@@ -181,6 +181,25 @@ extension API.PackageController.GetRoute.Model {
             return .empty
         }
     }
+    
+    func forkedListItem() -> Node<HTML.ListContext> {
+        if let forkedFromURL {
+            let repoURLNode: Node<HTML.BodyContext> = .a(
+                .href(forkedFromURL),
+                .text("repository")
```

Comment:
Complete outsider to conversations, so if this is the agreed solution then just ignore this.

It would feel a bit nicer to me if we were able to display the original author name here perhaps? "Forked from Apple" as an example.

Reply:
You took my words 😂

In discussions we talked about having it as "Forked from owner/repo.", which I think would be fine.

If we wanted to really push the boat out and be really extra, I could see this:

* If the original repo isn't on SPI, it says "Forked from owner/repo." and links to GitHub.
* If the original repo is on SPI and the package name (not repo name) is the same, it says "Forked from Original Owner Name" using the owner name that we already have. Thanks James for this suggestion.
* If, in addition to that, the package name has changed, it says "Forked from Original Package Name by Original Owner Name"

That'd be the absolute belt and braces implementation!

Reply:
Let's go for the absolute belt and braces, I'll update the PR to add those.

Reply:
Thanks for making these changes! This is really close now. Just a couple of tweaks:

1. In the case where we have a renamed package, the link looks good but let’s split the link into two so there’s a link to the package first, then the word “by” which isn’t a link (this will help visually distinguish the package name from the owner name), then a link to the owner’s page. Like this:

![Screenshot 2024-09-12 at 12 52 37@2x](https://github.com/user-attachments/assets/fca94611-8bf5-4140-9fd3-5ae816d57a10)

---

2. I think it looks slightly odd where the package is *not* renamed. In some instances it's fine, but examples like [XibLoc](http://swiftpackageindex.com/Frizlab/XibLoc) look odd and it’s not immediately obvious what “happn” is. Let’s slightly adjust this (and make it simpler) by including both the package name *and* the owner name in both cases, even if the package has not been renamed.

So. This:

![Screenshot 2024-09-12 at 12 44 06@2x](https://github.com/user-attachments/assets/aa10c224-0215-44ec-92df-9b99f1068832)

Becomes this:

![Screenshot 2024-09-12 at 12 45 59@2x](https://github.com/user-attachments/assets/d9ce3c0d-b91b-4949-a532-f2741e8db853)

As mentioned above, this should also separate the one link into two. The first being a link to the package and the second being a link to the author. The good news is that we’ve also just reduced the number of possible code paths from three to two.

---

3. Finally, I think there’s a crossed wire with the links to GitHub. When I said “Forked from owner/repository” I meant to replace the “repository” link with a link that says the owner and the repository variables, not the words.

So, for example. This:

![Screenshot 2024-09-12 at 12 55 18@2x](https://github.com/user-attachments/assets/62a487f2-bb0d-4f58-99d0-40105429eed0)

Becomes this:

![Screenshot 2024-09-12 at 12 56 45@2x](https://github.com/user-attachments/assets/6bfd99d2-32df-429d-9985-af3fdd9dce9f)